extensions/ql-vscode/src/view/results/resultsView.css

```diff
@@ -144,3 +144,12 @@ td.vscode-codeql__path-index-cell {
 .vscode-codeql__location-cell {
   text-align: right !important;
 }
+
+.vscode-codeql__taxa-cell {
+  text-align: left !important;
+}
+
+.vscode-codeql__taxa-cell-div {
```

Comment:
Would it make sense to call this `contents` instead of `div` since we're not dependent on this being a `div`?

Reply:
Done